chia/cmds/wallet_funcs.py

```diff
@@ -482,12 +482,18 @@ async def print_balances(args: dict, wallet_client: WalletRpcClient, fingerprint
         else:
             print(f"Balances, fingerprint: {fingerprint}")
         for summary in summaries_response:
+            asset_id = summary["data"]
             wallet_id = summary["id"]
             balances = await wallet_client.get_wallet_balance(wallet_id)
             typ = WalletType(int(summary["type"]))
             address_prefix, scale = wallet_coin_unit(typ, address_prefix)
-            print(f"Wallet ID {wallet_id} type {typ.name} {summary['name']}")
-            print(f"   -Total Balance: {print_balance(balances['confirmed_wallet_balance'], scale, address_prefix)}")
+            total_balance: str = print_balance(balances["confirmed_wallet_balance"], scale, address_prefix)
+            print(f"{summary['name']}: {total_balance}")
+            print(f"   -Wallet ID: {wallet_id}")
+            print(f"   -Type: {typ.name}")
+            if len(asset_id) > 0:
+                print(f"   -Asset ID: {asset_id}")
+            print(f"   -Total Balance: {total_balance}")
```

Comment:
Duplicating the total balance seems a little bit funny, but it also seems nice to have it in each place it is shown now.  Maybe just show it first in the list like below?  I'm not sure.  Maybe even a blank line between each wallet to help break it up?

```
Chia Wallet:
   -Total Balance: 0.000299999996 xch (299999996 mojo)
   -Pending Total Balance: 0.000299999996 xch (299999996 mojo)
   -Spendable: 0.000299999996 xch (299999996 mojo)
   -Wallet ID: 1
   -Type: STANDARD_WALLET

Spacebucks:
   -Total Balance: 1.234  (1234 mojo)
   -Pending Total Balance: 1.234  (1234 mojo)
   -Spendable: 1.234  (1234 mojo)
   -Wallet ID: 2
   -Type: CAT
   -Asset ID: 78ad32a8c9ea70f27d73e9306fc467bab2a6b15b30289791e37ab6e8612212b100
```

Reply:
Updated